dev

```diff
@@ -129,6 +129,8 @@ init() {
 init_in_docker() {
     require_in_docker
     wait_for_db
+    # make migrations
+    python manage.py makemigrations
```

Comment:
This is wrong. Creating migrations should always be conscious decision by the developer.